src/components/Map/Map.tsx

```diff
@@ -0,0 +1,23 @@
+'use client';
+import MapWrapper from './MapWrapper';
+import { MapContextProvider } from './contexts/MapContextProvider';
+import { Place } from './types';
+/**
+ * @property {onClose} onClose  - overlay를 닫는 함수입니다. hooks에 있는 useOverlay를 이용해 맵 overlay를 닫을 수 있습니다.
+ * @property {onClick} onClick  - 맵에서 장소를 선택했을 때 실행되는 함수입니다. 선택한 장소의 데이터를 인자로 받습니다.
+ */
```

Comment:
혹시 주석에 있는 내용을 아래 사진처럼 컴포넌트 사용시에 힌트를 공유하려는 의도로 작성하신 주석이 맞으실까요? 위와 같은 방식으로 주석을 쓰면 `MapProps` 라는 타입에다가 커서를 올려야지만 힌트가 제공되는 방식이더라구요. 그래서 목적이 컴포넌트 시에 힌트 제공이시라면 아래와 같은 JSDOC 작성 방식을 추천드립니다

```suggestion
type MapProps = {
  /** overlay를 닫는 함수입니다. hooks에 있는 useOverlay를 이용해 맵 overlay를 닫을 수 있습니다.  */
  onClose: () => void;
  /** 맵에서 장소를 선택했을 때 실행되는 함수입니다. 선택한 장소의 데이터를 인자로 받습니다. */
  onClick: (data: Place | undefined) => void;
} & React.HTMLAttributes<HTMLDivElement>;
```
![image](https://github.com/YAPP-Github/22nd-Web-Team-1-Web/assets/38908080/271cb8fc-c0dc-47d7-b7dd-6a1c581bdaa7)